packages/context/src/resolver.ts

```diff
@@ -309,7 +309,7 @@ export function resolveInjectedProperties(
     (properties[p] = v);
 
   for (const p in injectedProperties) {
-    const injection = injectedProperties[p];
+    let injection = injectedProperties[p];
```

Comment:
I find this change suspicious, `injection` is not changed after initialization. Is it possible to revert this line please?